### PR TITLE
Add rewards/:block endpoint for hotspots/accounts

### LINKED ISSUE #401: 'Rewards for an Account' request returns empty data and no cursor
request: https://api.helium.io/v1/accounts/:address/rewards

Sample return:
```
{
   'data': []
}
```

### PR DESCRIPTION
**Pull request by madninja**
Fixes https://github.com/helium/blockchain-http/issues/401

### CODE REVIEW

File: docs/api/blockchain/accounts.mdx [hunk 1]
```diff
@@ -1114,6 +1114,20 @@ Returns reward entries by block and gateway for a given account in a timeframe.
 Timestamps are given in ISO 8601 format. The block that contains the `max_time`
 timestamp is **excluded** from the result.
 
+The result will be a list of rewards between `min_time` and `max_time` both
+given in UTC. Both default to "now" which means that at least one of the two
+parameters is required.
+
+The result includes a `type` field which is the type activity that generated the
```
Comment (danielcolinjames) [inline docs/api/blockchain/accounts.mdx]:
```suggestion
The result includes a `type` field which is the type of activity that generated the
```

File: docs/api/blockchain/accounts.mdx [hunk 2]
```diff
@@ -1128,39 +1142,89 @@ _Path Parameters_
...
+
+---
+
+## Rewards in a Rewards Block for an Account
+
+```
+GET https://api.helium.io/v1/hotspots/:address/rewards/:block
+```
+
+Returns rewards for a given acount for a given block.
```
Comment (danielcolinjames) [inline docs/api/blockchain/accounts.mdx]:
```suggestion
Returns rewards for a given account for a given block.
```

**Review by danielcolinjames — CHANGES_REQUESTED** [review]
looks good! just a few small typos I noticed (and "rewads" in the title of this PR 🙂 )

**Review by danielcolinjames — APPROVED** [review]

**Review by matthewcarlreetz — APPROVED** [review]
LGTM!

**Pull request merged**